docs/en/cloud/security/aws-privatelink.md

```diff
@@ -17,7 +17,7 @@ AWS PrivateLink is only available in ClickHouse Cloud Production services. Devel
 Before you get started you will need:
 
 1. An AWS account.
-1. An API key with the necessary permissions to create and manage private links.
+1. A [ClickHouse Cloud API key](https://clickhouse.com/docs/en/cloud/manage/openapi) with the necessary permissions to create and manage private links.
```

Comment:
AFAIK we have only 2 roles(atm): developer and admin, shall we mention explicitly to create key w/ admin access?

Reply:
That makes sense to me, @konsta-danyliuk , what do you think?

Reply:
@natalyjazzviolin , yep, this seems more straightforward.

@tsolodov, If the Dev key type doesn't have sufficient permissions for Private Links set up, we should include the key type in the doc.